.pullapprove.yml

```
@@ -684,6 +684,8 @@ groups:
     conditions:
       - author in ["angular-robot"]
     reviewers:
+      users:
+        - alan-agius4
```

```suggestion
        - ~alan-agius4
```

Can we switch to this so that you don't get them assigned as well?